### PR TITLE
fix(typeahead-dropdown): suggestion issue with typeahead dropdown

### PR DESCRIPTION
**Pull request by vikram-raj**
issue: https://github.com/openshiftio/openshift.io/issues/2533

### CODE REVIEW

**Comment by fabric8cd** [comment]
@vikram-raj fabric8/fabric8-ui:SNAPSHOT-PR-2501-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2501-fabric8-planner.badger.fabric8.io

**Comment by Raunak1203** [comment]
@vikram-raj Long iteration text it shows like this in the dropdown:
screenshot from 2018-03-15 11-24-48

**Comment by vikram-raj** [comment]
> @vikram-raj Long iteration text it shows like this in the dropdown:

resolved it in https://github.com/fabric8-ui/fabric8-planner/pull/2501/commits/4be6c1fb014d23f656acb97b04b8d911b2c32cd7

**Comment by fabric8cd** [comment]
@vikram-raj fabric8/fabric8-ui:SNAPSHOT-PR-2501-10 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2501-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@vikram-raj fabric8/fabric8-ui:SNAPSHOT-PR-2501-11 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2501-fabric8-planner.badger.fabric8.io

**Comment by vikram-raj** [comment]
> As discussed, Could you please modify the tests

@Raunak1203  Done https://github.com/fabric8-ui/fabric8-planner/pull/2501/commits/5efca3de283983745fe3fd33291d511a64e538ad

File: src/tests/specs/smoke/smokeTest.spec.ts [hunk 1]
```diff
@@ -79,19 +79,27 @@ describe('Planner Smoke Tests:', () => {
...
+    await planner.quickPreview.addIteration(c.dropdownIteration2);
     expect(await planner.quickPreview.hasIteration(c.iteration2)).toBeTruthy();
     await planner.quickPreview.close();
 
     await planner.workItemList.clickWorkItem(c.workItemTitle3);
     expect(await planner.quickPreview.hasIteration(c.iteration2)).toBeTruthy();
     await planner.quickPreview.close();
+
+    await planner.workItemList.clickWorkItem(c.workItemTitle3);
+    await planner.quickPreview.typeaHeadSearch('z');
```
Comment (Raunak1203) [inline src/tests/specs/smoke/smokeTest.spec.ts]: @vikram-raj could you add this 'z' in constant.ts

File: src/tests/specs/smoke/smokeTest.spec.ts [hunk 2]
```diff
@@ -79,19 +79,27 @@ describe('Planner Smoke Tests:', () => {
...
+    expect(await planner.quickPreview.iterationDropdown.menu.getTextWhenReady()).toBe('No matches found.');
+    await planner.quickPreview.iterationCancelButton.click();
```
Comment (Raunak1203) [inline src/tests/specs/smoke/smokeTest.spec.ts]: instead of click , please use clickWhenReady()

File: src/tests/specs/smoke/smokeTest.spec.ts [hunk 3]
```diff
@@ -79,19 +79,27 @@ describe('Planner Smoke Tests:', () => {
...
+    await planner.quickPreview.iterationDropdown.click();
```
Comment (Raunak1203) [inline src/tests/specs/smoke/smokeTest.spec.ts]: same clickWhenReady()

**Comment by fabric8cd** [comment]
@vikram-raj fabric8/fabric8-ui:SNAPSHOT-PR-2501-12 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2501-fabric8-planner.badger.fabric8.io

File: src/tests/specs/smoke/smokeTest.spec.ts (hunk 1 above)
Comment (vikram-raj) [reply src/tests/specs/smoke/smokeTest.spec.ts]: @Raunak1203 Done.

File: src/tests/specs/smoke/smokeTest.spec.ts (hunk 2 above)
Comment (vikram-raj) [reply src/tests/specs/smoke/smokeTest.spec.ts]: @Raunak1203 Done.

File: src/tests/specs/smoke/smokeTest.spec.ts (hunk 3 above)
Comment (vikram-raj) [reply src/tests/specs/smoke/smokeTest.spec.ts]: @Raunak1203 Done.

**Comment by fabric8cd** [comment]
@vikram-raj fabric8/fabric8-ui:SNAPSHOT-PR-2501-13 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2501-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@vikram-raj fabric8/fabric8-ui:SNAPSHOT-PR-2501-20 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2501-fabric8-planner.badger.fabric8.io

**Pull request merged**